zookeeper-server/src/main/java/org/apache/zookeeper/cli/RestoreCommand.java

```diff
@@ -136,26 +169,45 @@ public RestoreCommand() {
 
   @Override
   public String getUsageStr() {
-    return "Usage: RestoreFromBackupTool  " + RESTORE_CMD_STR + " " + OPTION_STR + "\n    "
+    return "Usage: RestoreFromBackupTool  " + RESTORE_CMD_STR + " " + OPTION_STR
+        + "\n    Options for both offline restoration and spot restoration:\n    "
         + OptionFullCommand.RESTORE_ZXID
-        + ": the point to restore to, either the string 'latest' or a zxid in hex format. Choose one between this option or "
-        + OptionFullCommand.RESTORE_TIMESTAMP
-        + ", if both are specified, this option will be prioritized\n    "
+        + ": the point to restore to, either the string 'latest' or a zxid in hex format. "
+        + "Choose one between this option or " + OptionFullCommand.RESTORE_TIMESTAMP
+        + ", if both are specified, this option will be prioritized. "
+        + "Required for both offline restoration and spot restoration.\n    "
         + OptionFullCommand.RESTORE_TIMESTAMP
         + ": the point to restore to, a timestamp in long format. Choose one between this option or "
-        + OptionFullCommand.RESTORE_ZXID + ".\n    " + OptionFullCommand.BACKUP_STORE
-        + ": the connection information for the backup store\n           For GPFS the format is: gpfs:<config_path>:<backup_path>:<namespace>\n    "
-        + OptionFullCommand.SNAP_DESTINATION
-        + ": local destination path for restored snapshots\n    "
-        + OptionFullCommand.LOG_DESTINATION + ": local destination path for restored txlogs\n    "
-        + OptionFullCommand.TIMETABLE_STORAGE_PATH
-        + ": Needed if restore to a timestamp. Backup storage path for timetable files, for GPFS the format is: gpfs:<config_path>:<backup_path>:<namespace>, if not set, default to be same as backup storage path\n    "
+        + OptionFullCommand.RESTORE_ZXID
+        + ". Required for both offline restoration and spot restoration.\n    "
+        + OptionFullCommand.BACKUP_STORE
+        + ": the connection information for the backup store\n           "
+        + "For GPFS the format is: gpfs:<config_path>:<backup_path>:<namespace>\n           "
+        + "Required for both offline restoration and spot restoration.\n    "
+        + OptionFullCommand.SNAP_DESTINATION + ": local destination path for restored snapshots. "
+        + "Required for offline restoration.\n    " + OptionFullCommand.LOG_DESTINATION
+        + ": local destination path for restored txlogs. "
+        + "Required for both offline restoration.\n    " + OptionFullCommand.TIMETABLE_STORAGE_PATH
```

Comment:
Is this a typo? `Required for both offline restoration.`

Reply:
Typo: required for (only) offline restoration.